src/pug/blog/gyro_ratcheting.pug

```diff
@@ -0,0 +1,162 @@
+
+  p This last one is not a metaphor but real: You have a controller with a very imprecise gyro sensor, not good enough for doing full scale movements and fine moments with the same sensitivity. So you configure the #[em gyro for fine aiming only, and the thumbstick for large movements]. This works ok as a workaround, and you get familiar with it after many hours. Later, someone (Input Labs) is giving you the designs for a gyro controller that can perfectly do both full scale and fine movements. Do you still want to use the thumbstick as a workaround?
+  hr
+
+  h3 Rationale for focusing our research on gyro ratcheting
+  p At Input Labs we thought that it makes much more sense to research and develop solutions to remove the limitations of gyro #[em ratcheting as a generalist input method], compared with the shortcomings of flick stick, for all the reasons mentioned above:
+  ul
+    li Softer learning curve.
+    li Simplicity.
+    li Accessibility.
+    li More potential applications.
+    li Higher performance on restricted mobility scenarios.
+    li Higher performance on competitive scenarios.
+
+  table.manual.hdiv
+    tr
+      th Limitation
+      th Solution
+    tr
+      td Lack of accuracy, linearity and responsiveness when employing gyro both both large and fine movements
+      td Dual gyro, carefully tuned firmware
+    tr
+      td Engage and disengage gyro while being able to use other buttons
+      td Touch surface embedded within the ABXY buttons
+    tr
+      td Required software configuration
+      td The controller identifies itself as a HID mouse
+
+  p As a follow up to these developments, we decided to completely #[em remove the secondary thumbstick], but that is a long story better suited for its own blog entry.
+  p Does this means we are never going to use flick stick? No, #[em we may implement flick stick as an option in future updates] of the Alpakka firmware, mapped to the 8-direction switch, this of course would force the turns to snap in increments of 45 degrees. That option would not be enabled by default, but may be useful for some people, while they transition from flick stick to "HD" ratcheting, or just because accessibility reasons.
+  hr
+
+  h3 Conclusion
+  p With the current performance of the Alpakka controller reference design, we believe we achieved the goal of making gyro ratcheting not just an enthusiast/experimental input method, but a first-class citizen generalist input method. Several #[+a('talking_about_us') independent reviewers] seems to agree. We could even say that the Alpakka controller is the beginning of the #[em third generation] of gyro controllers.
```

Comment:
"citizen _for_ ..."
"seems" => "seem" => "agree"
"We consider the Alpakka the first device of a third generation of gyro controllers"